src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswerOption.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { memo, useEffect, useState } from 'react';
+import { isUndefined } from 'lodash-es';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Col, Collapsible, Icon, IconButton, Form, Row,
+} from '@edx/paragon';
+import { AddComment, Delete } from '@edx/paragon/icons';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+import { actions } from '../../../../../data/redux';
+import { answerOptionProps } from '../../../../../data/services/cms/types';
+
+export const AnswerOption = ({
+  answer,
+  hasSingleAnswer,
+  // injected
+  intl,
+  // redux
+  deleteAnswer,
+  updateAnswer,
+}) => {
+  const [isFeedbackVisible, setIsFeedbackVisible] = useState(false);
```

Comment:
If you look at files like  `VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/hooks.js` you'll see we use a sneaky method to make state very easy to test.